hr_public_holidays/security/ir.model.access.csv

```diff
@@ -0,0 +1,5 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_hr_holidays_public_user,access_hr_holidays_public,model_hr_holidays_public,base.group_user,1,0,0,0
+access_hr_holidays_public_manager,access_hr_holidays_public,model_hr_holidays_public,hr.group_hr_manager,1,1,1,1
+access_hr_holidays_public_line_user,access_hr_holidays_public_line,model_hr_holidays_public_line,base.group_user,1,0,0,0
+access_hr_holidays_public_line_manager,access_hr_holidays_public_line,model_hr_holidays_public_line,hr.group_hr_manager,1,1,1,1
```

Comment:
base.group_hr_* no longer exist. They have moved to hr.group_hr_*. I suspect that you haven't noticed this in your tests because by default they run as the admin user. Before you fix this try adding a test that access the public holidays as a normal user of hr.group_hr_user. It should fail.